lib/screens/search_screen/bloc/github_search_event.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:equatable/equatable.dart';
+
+abstract class GithubSearchEvent extends Equatable {
+  const GithubSearchEvent();
+
+  @override
+  List<Object> get props => [];
+}
+
+class InitEvent extends GithubSearchEvent {}
+
+class OnTechnologySelected extends GithubSearchEvent {
```

Comment:
Rename `OnTechnologySelected` to `OnTechnologySelectedEvent`. It will be better readability